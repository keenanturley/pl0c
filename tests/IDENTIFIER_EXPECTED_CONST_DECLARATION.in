const = 7, n = 85;
var i,x,y,z,q,r;

begin
    x := m;
    y := n;
end.