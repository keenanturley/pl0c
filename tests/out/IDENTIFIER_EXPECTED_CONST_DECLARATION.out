Lexeme List:
28 9 3 7 17 2 n 9 3 85 18 29 2 i 17 2 x 17 2 y 17 2 z 17 2 q 17 2 r 18 21 2 x 20 2 m 18 2 y 20 2 n 18 22 19 

Lexeme List (Symbolic Representation):
constsym eqsym numbersym 7 commasym identsym n eqsym numbersym 85 semicolonsym varsym identsym i commasym identsym x commasym identsym y commasym identsym z commasym identsym q commasym identsym r semicolonsym beginsym identsym x becomessym identsym m semicolonsym identsym y becomessym identsym n semicolonsym endsym periodsym 

Error: Identifier expected in const declaration.
